element-hq/element-x-android
# Fix scaling of timeline images

## Type of change


## Content

The scaling behaviour now matches iOS:

- Fit content scaling, not Crop.
- Min/max height, not min/max aspect ratio. Changes the aspect ratio to take into account width, then height.

## Motivation and context

Fixes #1940.

## Screenshots / GIFs

In the PR.

## Tests

- Send an image with long height.
- Check how it looks in the timeline.

## Tested devices

- OS version(s): 11

## Checklist
